getting-started/clis/calicoctl/install.md

```diff
@@ -17,17 +17,336 @@ You can run `calicoctl` on any host with network access to the
 For step-by-step instructions, refer to the section that
 corresponds to your desired deployment.
 
-- [As a binary on a single host](#install-calicoctl-as-a-binary-on-a-single-host)
+<!--- Change download URL to latest release if user browsing master branch.  --->
+{%- if page.version == "master" -%}
+{% assign version = "latest/download" %}
+{% else %}
+{% assign version = "download/" | append: site.data.versions.first.components.calicoctl.version %}
+{% endif %}
 
-- [As a container on a single host](#install-calicoctl-as-a-container-on-a-single-host)
```

Comment:
These bullet points are embedded in the theme, I think this was for an older theme that is why I think it should be removed.

@caseydavenport